packages/esm-active-visits-app/src/pagination/usePaginationInfo.ts

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useConfig } from '@openmrs/esm-framework';
+
+export function usePaginationInfo(pageSize: number, totalItems: number, pageNumber: number, currentItems: number) {
+  const { t } = useTranslation();
+  const config = useConfig();
+
+  const pageSizes = useMemo(() => {
+    const numberOfPages = Math.ceil(totalItems / config?.activeVisits?.pageSize);
+    return [...Array(numberOfPages).keys()].map((x) => {
+      return (x + 1) * config?.activeVisits?.pageSize;
+    });
+  }, [pageSize, totalItems]);
+
+  const numberOfItemsDisplayed = useMemo(() => {
+    if (pageSize > totalItems) {
```

Comment:
Any reason for not using `usePagination` from framework?

Reply:
`usePagination` is used in the main component to handle results' pagination. This is used to mainly get the pageSizes array.

Reply:
Personally, I don't see the need for any of these computations.

Usually, page sizes are fixed. They don't align with the items. They don't care about the items.